Marketplace/fingerPaint/content/animatedBrushes/animatedBrushScript.js

```diff
@@ -0,0 +1,37 @@
+(function() {    
+    Script.include("animatedBrushesList.js");
+    var UPDATE_TIME = 33; //run at aproximatelly 30fps
+    var MIN_PLAY_DISTANCE = 6; //Minimum distance from player to entity in order to play animation
+    var self = this;
+    this.preload = function(entityID) {
+        print("After adding script 2 : " + JSON.stringify(Entities.getEntityProperties(entityID).color));
```

Comment:
Remove print statements or wrap in a debug catch. Only request necessary entity properties with getEntityProperties()